tests/teal/user_guide_docs_dynamic_scratch_var_demo_expected.teal

```diff
@@ -0,0 +1,17 @@
+#pragma version 6
```

Comment:
No requirement to, but be away that you are free to "mark-up" this file with explanatory comments. The `assert_new_v_old` is designed to still pass as long as each line of `new` is a prefix of the `old` and both files have the same number of lines.